docs/package/gluon-ebtables-source-filter.rst

```diff
@@ -6,8 +6,8 @@ ruleset to prevent unreasonable traffic entering the network via the nodes.
 Unreasonable means traffic entering the mesh via a node which source IP does
 not belong to the configured IP space.
 
-One may first check if there is a certain proportion of unreasonable traffic,
-before adding this package to the firmware image. Additional one should not
+You may first check if there is a certain proportion of unreasonable traffic,
+before adding this package to the firmware image. Additional you should not
```

Comment:
there should be an "y" - "Additionally"
even better: "Furthermore, you should [...]"